deploy/standalone/flux-deployment.yaml

```diff
@@ -4,11 +4,25 @@ metadata:
   name: flux
 spec:
   replicas: 1
+
+  # Having maxSurge 0 and maxUnavailable 1 means the deployment will update one
```

Comment:
But why is this important or necessary? (some explanation in the comment would be good)